next/components/forms/useFormSend.tsx

```diff
@@ -21,6 +21,30 @@ import { useFormModals } from './useFormModals'
 import { useFormSent } from './useFormSent'
 import { useFormState } from './useFormState'
 
+/**
+ * This hook controls the sending of the form. The logic is scattered across the app.
+ *
+ * Form send:
+ * 1. The user clicks on the "Odoslať" button.
+ * 2. All the conditions in `handleSendButtonPress` are checked. If any of it fails an appropriate modal is shown.
+ * 3. Confirmation modal is shown.
+ * 4. `sendFormMutate` is called. On success, we redirect to the success page. On error, we show an error.
+ *
+ *
+ * Form eID send:
+ * 1. The user clicks on the "Odoslať s eID" button.
+ * 2. All the conditions in `handleSendEidButtonPress` are checked. If any of it fails an appropriate modal is shown.
+ * 3. Confirmation modal is shown.
+ * 4. `saveConceptAndSendEidMutate` is called. This doesn't save the modal! This triggers the concept save and redirects to the eID login page.
```

Comment:
`This doesn't save the modal!` - probably a misstype, or I don't understand

Reply:
send the form :D